src/IP.cc

```diff
@@ -288,7 +288,7 @@ RecordVal* IPv6_Hdr::BuildRecordVal(VectorVal* chain) const
 			}
 
 		default:
-			reporter->Weird(fmt("unknown_mobility_type_%d", mob->ip6mob_type));
+			reporter->Weird(fmt("unknown_mobility_type", mob->ip6mob_type));
```

Comment:
Should this still be fmt() like that ?

```suggestion
			reporter->Weird("unknown_mobility_type", fmt("%d", mob->ip6mob_type));
```

Reply:
Thanks, I pushed a fix (this code path gets pre-processed out by default, so there was no compiler complaints).